src/python/pants/option/options_bootstrapper_test.py

```diff
@@ -3,103 +3,123 @@
 
 import os
 import unittest
+from functools import partial
 from textwrap import dedent
+from typing import Dict, List, Optional
 
 from pants.base.build_environment import get_buildroot
+from pants.option.option_value_container import OptionValueContainer
 from pants.option.options_bootstrapper import OptionsBootstrapper
 from pants.option.scope import ScopeInfo
 from pants.util.contextutil import temporary_dir, temporary_file, temporary_file_path
 
 
-class BootstrapOptionsTest(unittest.TestCase):
+class OptionsBootstrapperTest(unittest.TestCase):
 
-  def _do_test(self, expected_vals, config, env, args):
```

Comment:
This was removed in favor of inlined helper functions in the two test methods that were using this. Those tests now directly call `assert_bootstrap_options` (formerly `_test_bootstrap_options`), rather than using this wrapper.